tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -643,7 +643,7 @@ public function testSetDefaultFetchMode()
         $stmt->setFetchMode(\PDO::FETCH_NUM);
 
         $row = array_keys($stmt->fetch());
```

Comment:
It's not really a `$row` since it doesn't represent the selected row data. It's more of a `$keys`.